pkgs/top-level/all-packages.nix

```
@@ -25795,6 +25795,8 @@ in
 
   secretscanner = callPackage ../tools/security/secretscanner { };
 
+  semiphemeral = callPackage ../tools/social/twitter/semiphemeral { };
```

Sorry I didn't see this before.
```suggestion
  semiphemeral = callPackage ../tools/misc/semiphemeral { };
```